benchmarks/build.gradle

```diff
@@ -14,13 +14,13 @@ buildscript {
     }
     dependencies {
         classpath libraries.protobuf_plugin
-        classpath "me.champeau.gradle:jmh-gradle-plugin:0.2.0"
+        classpath libraries.jmh_gradle_plugin
     }
 }
-apply plugin: 'me.champeau.gradle.jmh'
+apply plugin: jmh_plugin_name
```

Comment:
Let's keep the full classname in-line and not use a variable. All plugins are slowly moving toward using package-like names. Using a variable for every plugin seems like it would be an anti-pattern.